test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Alias.Tests.ps1

```diff
@@ -155,6 +155,29 @@ Describe "Get-Alias DRT Unit Tests" -Tags "CI" {
             $returnObject[$i].Definition | Should -Be 'Get-Command'
         }
     }
+
+    # Below test was suggested by GitHub Copilot to aid with fixing issue #25616
```

Comment:
Please remove the comment ant all comments below for the new test - they did not add a value.